packages/components/src/components/validate/validate.component.ts

```diff
@@ -0,0 +1,284 @@
+import type { CSSResultGroup } from 'lit';
+import { html } from 'lit';
+import { property, queryAssignedElements, state } from 'lit/decorators.js';
+import componentStyles from '../../styles/component.styles.js';
+import SynergyElement from '../../internal/synergy-element.js';
+import { watch } from '../../internal/watch.js';
+import SynAlert from '../alert/alert.component.js';
+import {
+  arraysDiffer,
+  getEventNameForElement,
+  isBlurEvent,
+  isInvalidEvent,
+  normalizeEventAttribute,
+  onConverter,
+} from './utility.js';
+import styles from './validate.styles.js';
+
+/**
+ * @summary Validate is a helper that may be used to wrap
+ * synergy input fields and forms to provide validation message.
+ *
+ * @dependency syn-alert
+ *
+ * @slot - The input field or form element to be validated.
+ * Avoid slotting in more than one element, as subsequent ones will be ignored.
+ *
+ * @csspart base - The component's base wrapper.
+ * @csspart input-wrapper - The container that wraps the input field.
+ * @csspart alert - The syn-alert that is shown when inline is true.
+ * @csspart alert__base - The container that wraps the alert.
+ * @csspart alert__message - The container that wraps the alert message.
+ * @csspart alert__icon - The container that wraps the alert icon.
+ */
+export default class SynValidate extends SynergyElement {
+  static styles: CSSResultGroup = [componentStyles, styles];
+
+  static dependencies = {
+    'syn-alert': SynAlert,
+  };
+
+  controller = new AbortController();
+
+  @queryAssignedElements() private slottedChildren: HTMLElement[];
+
+  @state() validationMessage = '';
+
+  /** Show the validation message underneath the element, using a syn-alert */
+  @property({ reflect: true, type: Boolean }) inline = false;
+
+  /** Do not show the error icon when using inline validation */
+  @property({ attribute: 'hide-icon', reflect: true, type: Boolean }) hideIcon = false;
+
+  /**
+   * Defines the events that trigger the validation.
+   * `invalid` will always automatically be included.
+   * You may also use the `live` keyword to validate on every input change.
+   * `live` will make sure to listen to the `invalid`, `input` and `blur` events.
+   * @example ```html
+   * <!-- Validate on invalid and change events (invalid, change) -->
+   * <syn-validate on="invalid change"></syn-validate>
+   *
+   * <!-- Validate on live events (invalid, blur, input)-->
+   * <syn-validate on="live"></syn-validate>
+   *
+   * <!-- Validate on live and custom events (invalid, blur, input, focus, change) -->
+   * <syn-validate on="live focus change"></syn-validate>
+   * ```
+   */
+  @property({
+    converter: onConverter(),
```

Comment:
I would remove the attribute converter and just work completely with strings. The conversion can then be done on the specific places, where it is needed (I think it is only needed in the `handleListenerChange` line 87)

Because there are some strange behaviors / something is not completely correct.

Following html results in following DOM structure:

```html
<syn-validate>
      <syn-input></syn-input>
</syn-validate>

<syn-validate on="change blur">
      <syn-input></syn-input>
</syn-validate>
```

![image](https://github.com/user-attachments/assets/649825c6-3d18-4a46-bebb-38ab5fd377b6)

So the array is shown in the dom (see the comma separation).


Also the arguments of the `handleListenerChange`  are never from type `string []` but always from type `string`. No matter if changing it via attribute or via property (see both gifs). I am logging the `typeof` of the arguments 
![image](https://github.com/user-attachments/assets/046f40dd-7307-45e0-91e0-fd93cde7aea1)

![validate-attribute-change](https://github.com/user-attachments/assets/1fab5d0b-0e17-4b82-a593-57e126289400)

![validate-change-prop](https://github.com/user-attachments/assets/4f9f41a9-1282-40b8-aaba-ca37e814f61c)

Reply:
I think that I might have messed up the `fromAttribute` conversion. Makes sense to fix this

Reply:
This is now always a string. Also makes it more convenient to set in Angular or React, because both would have needed something like `<SynValidate on={['a', 'b']} />`, which was kind of arkward.